web-ng/src/app/services/router/router.service.ts

```diff
@@ -69,19 +69,17 @@ export class RouterService {
     this.observationId$ = fragmentParams$.pipe(
       map(params => params.get(RouterService.OBSERVATION_ID_FRAGMENT_PARAM))
     );
-    this.sideNavContentType$ = fragmentParams$.pipe(
-      map(params => { 
-        console.log(params);
-        console.log(params.get(RouterService.OBSERVATION_ID_FRAGMENT_PARAM));
+    this.sideNavContentMode$ = fragmentParams$.pipe(
+      map(params => {
         if (params.get(RouterService.OBSERVATION_ID_FRAGMENT_PARAM)) {
-          return SideNavContentType.OBSERVATION;
+          return SideNavContentMode.OBSERVATION;
```

Comment:
Can you extract this logic in a static function that maps params to SideNavContentMode? It would make this pipe a little more friendly to read since it would separate the Rx plumbing (no pun intended) from the conversion logic.

Reply:
done